src/background.ts

```diff
@@ -879,10 +942,25 @@ ipcMainHandle("VALIDATE_ENGINE_DIR", (_, { engineDir }) => {
   return engineManager.validateEngineDir(engineDir);
 });
 
-ipcMainHandle("RESTART_APP", async (_, { isMultiEngineOffMode }) => {
-  appState.willRestart = true;
-  appState.isMultiEngineOffMode = isMultiEngineOffMode;
-  win.close();
+ipcMainHandle("RELOAD_APP", async (_, { isMultiEngineOffMode }) => {
+  win.hide(); // FIXME: ダミーページ表示のほうが良い
+
+  // FIXME: 同じようなURLだとスーパーリロードされないことがあるので一度ダミーページを読み込む
```

Comment:
Vue側でrouterのパラメータを監視すればいける…かも？未検証。

Reply:
いったんこのままで行こうかなと思いました！
ちなみにwatchしておいたら勝手にスーパーリロードになるかも、という意図でしょうか？ 👀

Reply:
あ、今見たら完全に別問題ですね…ごめんなさい

https://www.electronjs.org/docs/latest/api/browser-window#winreload
これはダメだったんでしたっけ？

Reply:
あ、試してないです！
というのもURLを変えてloadしてすぐreloadするのか、reloadしてすぐURLを変えてloadするのかどっちが妥当なのか考えて、まあダミーページに遷移したほうが確実な気がしたのでそうした感じです。

Reply:
なるほど。URLロード->リロードで動くとも思いますが、ダミーページが確実ですね。
となるとFIXMEじゃなくてもいいような気がします。

Reply:
あれ本当ですね、ここのコメントちょっとおかしかったです。。
いつか直す時のためにここにコメント残しときます。。

```suggestion
  // NOTE: 同じようなURLだとスーパーリロードされないことがあるので一度ダミーページを読み込む
  // FIXME: ダミーページを読み込まなくても良いようにする
```